web/styles/compose.css

```diff
@@ -1178,10 +1170,12 @@ textarea.new_message_textarea,
         }
     }
 
-    .fa-chevron-down {
-        padding-left: 5px;
-        color: hsl(0deg 0% 58%);
-        font-weight: lighter;
+    #compose_select_recipient_widget {
+        .zulip-icon-chevron-down {
```

Comment:
without this, the dropdown area will be shorter and look like compressed.
without this.
![image](https://github.com/zulip/zulip/assets/100200105/f9ef6e4c-aa6c-4885-9225-ae01c15b5f32)
with this 
![image](https://github.com/zulip/zulip/assets/100200105/7d453a2b-511a-4a61-9fde-da8f0600731c)

Reply:
already addressed in e6718369d6dc759fc72a25d41a57101433d5596e.